data/data.go

```diff
@@ -21,17 +21,16 @@ func GetCluster(
 	c InstalledData,
 	i ClusterID,
 	v AvailableComponentVersion,
-	secretGetterCreator KubeSecretGetterCreator,
 ) (models.Cluster, error) {
 
 	// Populate cluster object with installed components
 	cluster, err := GetInstalled(c)
 	if err != nil {
 		return models.Cluster{}, err
 	}
-	err = AddUpdateData(&cluster, v, secretGetterCreator)
+	err = AddUpdateData(&cluster, v)
 	if err != nil {
-		return models.Cluster{}, err
+		log.Println("unable to decorate cluster data with available updates data")
```

Comment:
2 things:

1. why not return the error anymore?
2. this log message should include the error in it